src/components/NewsItem.vue

```diff
@@ -66,11 +66,13 @@ export default {
 
 .card {
   position: relative;
+  z-index: 1;
+  flex-grow: 1;
+  width: 100%;
   background-position: center;
   background-size: cover;
   transition: .4s all;
-  z-index: 1;
-  width: 100%;
+  margin-bottom: 0.75rem;
```

Comment:
A better solution here would be to have `$base-line-height / 2` as we want to avoid custom margin/padding for elements so that it keeps in line with our grid system that we have developed.